src/main/java/seedu/address/model/Model.java

```diff
@@ -76,12 +78,28 @@ public interface Model {
      */
     void setPerson(Person target, Person editedPerson);
 
+    //    void hasTransaction(Transaction transaction);
```

Comment:
To update in future iteration:
Implement transaction methods and relation update methods.